SenecaAurelius/liftoff-project
# Initial setup

This initial commit should set up the SpringBoot/Thymeleaf project for our app, and it contains the information to set up a persistent database locally using SQL.

To test before merging, I'd recommend:
1) checkout the initial-setup branch
2) make sure you set up a "liftoff-project" schema in MYSQL
3) see if when you run the Gradle bootRun task the application starts without any errors

Be sure to bring up issues here!
